package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-if",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Changing the patch version for a breaking change is something the community has embraced, and I'm glad we're embracing it too!